unstructured/ingest/connector/jira.py

```diff
@@ -323,18 +296,42 @@ def _output_filename(self):
             Path(self.partition_config.output_dir) / self.grouping_folder_name / output_file
         ).resolve()
 
+    def update_source_metadata(self, **kwargs) -> None:
+        exists = bool(self.issue)
+        if not exists:
+            self.source_metadata = SourceMetadata(
+                exists=exists,
+            )
+            return
+        version = None
+        if len(self.parsed_fields["versions"]) > 1:
```

Comment:
Otherwise, we won't be making use of the version info when there is only one version element in the version list

If we do not assign version info before there are at least two versions of a doc, please ignore this suggestion

```suggestion
        if len(self.parsed_fields["versions"]) >= 1:
```